scripts/index.ts

```diff
@@ -22,6 +23,9 @@ async function run(steps: string[]) {
       continue;
     }
     switch (step) {
+      case 'check-install':
+        await checkInstall();
```

Comment:
Looks like all these `await`s in this `switch` statement are just "fire-and-forget" things. Do we need `await`?

Reply:
In this place we need `await`, since this switch statement takes place in a for loop iterating over "step" names. It's true that not every usage of `await` is needed, but for consistency's sake I prefer using it over *Sync operations.

Reply:
Sorry I missed the for loop.